packages/base32-address/index.js

```diff
@@ -208,9 +211,13 @@ export const randomBase32Address = (...args) => {
   if (netId === undefined) netId = 1029
 
   let hexAddress = randomHexAddress()
-  if (type === 'user') hexAddress = hexAddress.replace(/0x./, '1')
-  else if (type === 'contract') hexAddress = hexAddress.replace(/0x./, '8')
-  else if (type === 'builtin') hexAddress = hexAddress.replace(/0x./, '0')
+  if (type === 'user') hexAddress = hexAddress.replace(/^0x./, '1')
```

Comment:
the code block is used to remove `0x` prefix.
can we use `randomHexAddress` with a param such as `prefix` to return the address with `0x` or not? And default `prefix` value is `true`.

Reply:
fixed this with stripHexPefix